plugins/bundle/plugin.go

```diff
@@ -385,6 +385,17 @@ func (p *Plugin) newDownloader(name string, source *Source) Loader {
 		// wrap the callback to include the name of the bundle that was updated
 		p.oneShot(ctx, name, u)
 	}
+	if client.Config().Type == "oci" {
+		ociStorePath := "oci" // use current folder oci
```

Comment:
I don't understand this comment. Does it mean we're pulling things into the file system in folder `oci/`?

Reply:
I'll update the comment here to clarify this a bit. 
The OCI downloader needs a path for the local OCI store. If the `PersistenceDirectory` is configured it adds an `oci` folder in that path to store the downloaded files, but if the `PersistenceDirectory` is nil it will add the `oci` folder to the current directory.
 
I'm very open to suggestions here if we want to error out if the `PersistenceDirectory` is not configured or set a different path.

Reply:
> if we want to error out if the PersistenceDirectory is not configured or set a different path.

I think this would be a decent choice. We'll need to ensure that the docs are clear, but if the OCI downloader needs a place to put stuff, we should require PersistenceDirectory to be set. Would it ever read back from that directory over service restarts...? Is it used as a download cache of some sort? 🤔